src/components/FocusZone/FocusZone.tsx

```diff
@@ -575,4 +581,31 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
   }
 
+  private _isElementInput(element?: HTMLElement): boolean {
+    if (element.tagName && element.tagName.toLowerCase() === 'input') {
+      return true;
+    }
+  }
+
+  private _shouldInputLoseFocus(element: HTMLInputElement, isForward?: boolean) {
+      let selectionStart = element.selectionStart;
+      let selectionEnd = element.selectionEnd;
+      // This means that the input has text selected and we shouldn't lose focus.
+      if (selectionStart !== selectionEnd) {
+        return false;
+      } else {
+        let inputValue = element.value;
+
+        if (element) {
```

Comment:
why are whe checking if (element) right after we actually try to use it ? (in element.value, element.selectionStart, element.selectionEnd)

Lets make this check happen at the beginning of the method